core/src/utils/sanitization/index.ts

```diff
@@ -0,0 +1,129 @@
+/**
+ * Does a simple sanitization of all elements
+ * in an untrusted string
+ */
+
+export const sanitizeDOMString = (untrustedString: string | undefined): string | undefined => {
+  try {
+    if (typeof untrustedString !== 'string') { return untrustedString; }
+
+    const blockedTags = ['script', 'style', 'iframe', 'meta', 'link', 'object', 'embed'];
```

Comment:
probably can be moved to an array at the root level, so it's not evaluated everything it's called